chaos_wrapper.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+function rollback() {
+    # avoid commits from the same faulty PR
+    rb_commit=$(git log --author Chaos --format=format:%H | tail -n +2 | head -n 1)
```

Comment:
Security wise, there is a much deeper problem: as soon as somebody starts their own cron job, they can basically turn chaosbot into a... well, bot (listening to a C&C server). This is not very hard, as we saw with #77... You just need to hide a commit that contains some nefarious code behind a commit that reverts it. That way, your code is in the repo. If at any point you can manage to get the production server to git checkout and run your own copy, then you have backdoored chaosbot, kind of like [Ken Thompson's famous talk](http://www-inst.eecs.berkeley.edu/~cs161/fa08/papers/p761-thompson.pdf).